src/components/loggedOut/loggedOutWelcome.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react"
+import { Typography, Grid, Button } from "@mui/material"
+
+export default function welcomeMessage() {
+
+    //contains the welcome message for the header to the left
+    return (
+        <Grid item container xs={5} direction="column" p={5} rowGap={2} alignItems="center" color="#F2E8DE">
+
+            <Typography variant="h2" textAlign={'center'}>Find Your Perfect Mentor or Mentee with Mesh</Typography>
+            <Typography variant="body1" fontWeight={250} textAlign={'center'}>Unlock Your Potential with Expert Guidance and Support.</Typography>
+            <Button variant="contained"
+                sx={{
+                    background: '#68D391',
+                    textDecoration: 'uppercase',
+                    padding: '6px 16px',
+                    fontSize: '20px',
+                    lineHeight: '24px',
+                    maxWidth: '250px',
+                    minWidth: '120px',
+                    width: '30%',
+                    height: '50px',
+                    borderRadius: '15px'
+                }}>
```

Comment:
When hovering over the button, mui styling takes over and turns the button blue; for UX purposes, make the hover effect turn into a different shade of green.

Maybe something like this:
backgroundColor: "#0B7D66",
            fontFamily: "cocogoose",
            "&:hover": {
              backgroundColor: "#0A6B57",
            },

Reply:
I've added the hover coloring